docs/application/dotnet/guides/multimedia/webrtc.md

```diff
@@ -0,0 +1,509 @@
+    ```
+    > [!IMPORTANT]
+    > `ApplyAudioStreamPolicy()` or `Display` must be called inside of the callback set by `TrackAdded` event handler if you want to output the audio or video track from the remote peer to the local target device's audio device or video display.
+
+2. To get media packet handle which packs the audio or video data from a remote peer, use `AudioFrameEncoded` or `VideoFrameEncoded` event:
+
+    ```csharp
+    webRtc.AudioFrameEncoded += (s, e) =>
+    {
+        // Use media packet - copy it's data or pass it to another API.
+        // Unref MediaPacket after use.
+    }
+    webRtc.VideoFrameEncoded += (s, e) =>
+    {
+        // Use media packet - copy it's data or pass it to another API.
+        // Unref MediaPacket after use.
+    }
+    ...
+    await webRtc.StartAsync();
+    // After finishing negotiation, AudioFrameEncoded or VideoFrameEncoded event handler could be called if receiving audio/video data from the remote peer exists
+    ```
+
+3. To render sending audio/video data on the local target device, use `EnableAudioLoopback()` or `EnableVideoLoopback()`:
+
+    ```csharp
+    ...
+    var audioSource = new MediaMicrophoneSource();
+    var videoSource = new MediaCameraSource();
+
+    webRtc.AddSource(audioSource);
+    webRtc.AddSource(videoSource);
+
+    audioSource.EnableAudioLoopback(new AudioStreamPolicy(AudioStreamType.Media));
+    videoSource.EnableVideoLoopback(new Display(window));
+    ...
+    await webRtc.StartAsync();
+    ```
+
+4. To change display mode or display visibility, use `MediaStreamTrack.DisplayMode` or `MediaStreamTrack.DisplayVisible` property:
+
+    Three types of display mode exist, `WebRTCDisplayMode.LetterBox`, `WebRTCDisplayMode.OriginSize`, and `WebRTCDisplayMode.Full`.
+    These properties are also available for any MediaStreamTrack of video loopback.
+    In the following example code, it tries to change the mode to `WebRTCDisplayMode.OriginSize` after display is set and change the visibility in an event function:
+
+    ```csharp
+    webRtc.TrackAdded += (s, e) =>
+    {
+        if (e.MediaStreamTrack.Type == MediaType.Video)
+        {
+            // You should create or get window instance to display video.
+            e.MediaStreamTrack.Display = new Display(window);
+            e.MediaStreamTrack.DisplayMode = WebRTCDisplayMode.OriginSize;
+            e.MediaStreamTrack.DisplayVisible = true;
+        }
+    }
+    ...
+    await webRtc.StartAsync();
+    ```
+
+<a name="statistics_info"></a>
+## Get statistics information
+
+You can get the statistics information WebRTC.
+
+1. To get statistics information, use `WebRTC.GetStatistics()`. This method is possible in `WebRTCState.Playing` state.:
+    ```csharp
+    ...
+    // WebRTCState is changed to Playing
+    ...
+    var stats = webRtc.GetStatistics(WebRTCStatisticsCategory.All);
```

Comment:
Hello @hsgwon , I have a similar question here regarding the function. The function being used in the code here is different from the one mentioned in Line# 492 `WebRTC.GetStatistics()`. Which is the correct one?